sidecar/proxy/envoy/manager.go

```diff
@@ -565,60 +548,6 @@ func buildFS(ruleList []api.Rule, workingDir string) error {
 		return err
 	}
 
-	dirName, err := ioutil.TempDir(workingDir+runtimeVersionsPath, "")
-	if err != nil {
```

Comment:
can you just keep this as a separate function or comment it out, instead of removing the functionality?